tutorials/README.md

```diff
@@ -1,6 +1,6 @@
 <a href="https://explosion.ai"><img src="https://explosion.ai/assets/img/logo.svg" width="125" height="125" align="right" /></a>
 
-# 🪐 Project Templates: Tutorials (12)
+# 🪐 Project Templates: Tutorials (11)
```

Comment:
This changes seems like it belongs in a separate PR, is there a reason for having it here (that I am missing in this huge set of changes)?

Reply:
Not that I can recall - guess I updated the project summary docs when I updated the project docs to reflect changed step names. Should I revert these changes?

Reply:
@adrianeboyd Is this ok as-is or should these changes be reverted?

Reply:
Resolving as this was overwritten by changes in `master` anyway.